tests/sequence_sharding/run.sh

```diff
@@ -50,9 +50,11 @@ function run() {
     check_log_contains $WORK_DIR/worker1/log/dm-worker.log "task can't auto resume"
 
     # resume manually
+    # this operation may not return 3 `"result": true`, because worker may
+    # - response too slowly to resume, so resume still see old error and waitOperationOk will return early
```

Comment:
that's not a "declarative command", and I think the right way of `waitOperationOk` of resuming is resuming at least once and see that result. Our persistent data in etcd may not support this change